hive3/src/main/java/org/apache/iceberg/mr/hive/serde/objectinspector/IcebergDateObjectInspectorHive3.java

```diff
@@ -69,4 +69,12 @@ public Object copyObject(Object o) {
     }
   }
 
+  @Override
+  public LocalDate convert(Object o) {
+    if (o == null) {
+      return null;
+    }
+    Date date = (Date) o;
```

Comment:
nit of the nit: new line after if block 😄

Reply:
fixed